src/us/kbase/workspace/test/kbase/HandleAndBytestreamIntegrationTest.java

```diff
@@ -73,12 +73,12 @@
 import us.kbase.workspace.test.controllers.minio.MinioController;
 import us.kbase.workspace.test.controllers.shock.ShockController;
 
-public class HandleAndShockTest {
+public class HandleAndBytestreamIntegrationTest {
 
-	/* The Handle ID tests were written years before the Shock ID tests, and cover a lot
-	 * of cases that are general to both ID types, such as making sure all the methods that
-	 * allow retrieving an object set the permissions correctly. As such, the Shock ID tests
-	 * concentrate on testing cases that are unique to Shock IDs.
+	/* The Handle ID tests were written years before the Bytestream (nee Shock) ID tests,
```

Comment:
what is "nee shock"

Reply:
read as 'formerly known as shock'

https://www.merriam-webster.com/dictionary/n%C3%A9e